src/tls.cpp

```diff
@@ -170,6 +170,11 @@ TLSInfo_GetTidOffset(void)
   }
 #endif
 
```

Comment:
Let's add a small FIXME comment here, that we also need the `offset` for `__aarch64__`.